blockchain/types/tx_internal_data.go

```diff
@@ -55,6 +55,9 @@ const (
 	TxTypeBatch, _, _
 	TxTypeChainDataAnchoring, TxTypeFeeDelegatedChainDataAnchoring, TxTypeFeeDelegatedChainDataAnchoringWithRatio
 	TxTypeLast, _, _
+	TxTypeEthEnvelope = 0x78
+	TxTypeAccessList = 0x7801
+	TxTypeDynamicFee = 0x7802
```

Comment:
```suggestion
	TxTypeEthEnvelope = TxType(0x78)
	TxTypeAccessList  = TxType(0x7801)
	TxTypeDynamicFee  = TxType(0x7802)
```

Without type conversion, they are not `TxType`, but just `int`.